certbot-ci/certbot_integration_tests/certbot_tests/test_main.py

```diff
@@ -542,24 +542,11 @@ def test_wildcard_certificates(context):
     assert exists(join(context.config_dir, 'live', certname, 'fullchain.pem'))
 
 
-def test_ocsp_status_stale(context):
```

Comment:
I don't think we should entirely delete this test. I think it's useful for testing for "EXPIRED" and to a lesser extent "TEST_CERT" in `certbot certificates` output.

Neither of these things have to do with OCSP though so the function could use a better name and maybe a better approach for testing for this behavior. Assuming this test still works, I'd recommend leaving it as is for now.

Reply:
Well, I definitetly think that the current setup of this test is now overkill regarding its purpose. However I agree this should be done in a separate PR. We would certainly be able to make the same tests, but without embedding a full certbot config in the asset tests.

For now I revert the removal.